extensions/wikia/SASS/SassUtil.php

```diff
@@ -453,4 +455,19 @@ public static function colorNameToHex($colorName) {
 		}
 	}
 
+	/**
+	 * @desc Converts theme designer color names to hex
+	 *
+	 * @param $themeSettings
+	 * @return mixed
+	 */
+	public static function normalizeThemeColors($themeSettings) {
+		foreach (self::THEME_DESIGNER_COLOR_KEYS as $key) {
+			if( !preg_match(self::HEX_REG_EXP, $themeSettings[$key])) {
+				$themeSettings[$key] = self::colorNameToHex($themeSettings[$key]);
```

Comment:
I'm not sure how about invoking function, but there might be a need for spacing in the round brackets as well :)